conda/dev.yml

```
@@ -8,12 +8,12 @@ dependencies:
   # =================
   - python >=3.9
   - pip
-  - matplotlib-base
-  - mpas_tools
+  - matplotlib-base # TODO: Find constraint
+  - mpas_tools >=0.21.0
   - netcdf4
   - numpy >=2.0,<3.0
   - xarray >=2023.02.0
-  - xcdat
+  - xcdat # TODO: Find constraint
```

```suggestion
  - xcdat >=0.7.0,<1.0
```
Again, the lower bound is the version pinned in the last E3SM-Unified, the upper bound is the next major version, assuming that would likely have breaking changes.

@tomvothecoder, should this be tighter, e.g. 0.8.0? Often, minor versions have breaking changes for packages before the 1.0.0 release.  Is that true of xcdat?

---

xCDAT is becoming more stable with each new release. If there are breaking changes, we usually have deprecation warnings. However, I do think it is a good idea to use `xcdat >=0.7.0,<1.0` until a 1.0 is officially released.